src/lib/chips/chip-list.ts

```diff
@@ -347,6 +347,11 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
   }
 
   ngAfterContentInit() {
+    // Needs to happen on the next tick to avoid `changed after checked` errors.
+    Promise.resolve().then(() => {
```

Comment:
This only happens once, but wouldn't  this make more sense to move to the initializeSelection method so that the default is propagated to newly created chips (when the querylist changes)? Otherwise, couldn't you end up in a state where the initial chips are not selectable but newly added chips (after initialization) are?

Reply:
Good point. I've added some logic to sync the state to new chips.